src/collective/fingerpointing/subscribers/configure.zcml

```diff
@@ -97,4 +97,10 @@
       handler=".iterate_logger"
       />
 
+  <!-- Workflow -->
+  <subscriber
+      for="Products.CMFCore.interfaces.IActionSucceededEvent"
+      handler=".workflow_logger"
+      />
+
```

Comment:
yes, it works for both; our testing matrix uses Archetypes in Plone 4.3 and Dexterity in Plone 5.